src/App.css

```diff
@@ -6,11 +6,15 @@
   width: 100%;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
   font-size: calc(3px + 2vmin);
   color: rgb(37, 3, 3);
 }
 
+@media (min-width: 819px) {
+  .app {
+    justify-content: center;
+  }
+}
```

Comment:
Instead of writing breakpoints like this use the breakpoints given by Material UI.